projects/01-pair-matching/hadi-mobarra/webpack/src/index.js

```diff
@@ -0,0 +1,106 @@
+import './styles.css';
+
+// Fetch API to get the number of row and column
+async function chartNumberFunc() {
+  const chartNumber = await fetch('http://192.168.43.126:8000/playgame')
+    .then((response) => response.json())
+    .then((data) => data.dimension)
+    .catch((err) => (err));
```

Comment:
Please wait for fetch for some time and if there is no response, abort fetch and return a number yourself.